core/src/test/java/org/kiwix/kiwixmobile/core/page/history/viewmodel/HistoryStateTest.kt

```diff
@@ -25,56 +25,52 @@ import org.kiwix.kiwixmobile.core.page.historyItem
 import org.kiwix.kiwixmobile.core.page.historyState
 
 internal class HistoryStateTest {
-  @Test
-  internal fun `isInSelectionMode is true when item is selected`() {
-    assertThat(historyState(listOf(historyItem(isSelected = true))).isInSelectionState)
-      .isEqualTo(true)
-  }
-
-  @Test
-  internal fun `isInSelectionMode is false when no item is selected`() {
-    assertThat(historyState(listOf(historyItem(isSelected = false))).isInSelectionState)
-      .isEqualTo(false)
-  }
 
   @Test
-  internal fun `historyListItems returns history from all books when showAll is true`() {
+  internal fun `visiblePageItems returns history from all books when showAll is true`() {
```

Comment:
this shouldn't be the responsibility of HistoryState I think, showAll logic is in the parent, these tests should be focused around the date headers